packages/cozy-harvest-lib/src/components/KonnectorConfiguration/DataTab.jsx

```diff
@@ -56,7 +60,18 @@ export const DataTab = ({
       )}
       <LaunchTriggerCard initialTrigger={trigger} disabled={isInMaintenance} />
       {documentSaveFolderId && (
-        <DocumentsLinkCard folderId={documentSaveFolderId} />
+        <AppLinkCard
+          slug="drive"
+          path={`/files/${documentSaveFolderId}`}
+          icon="file"
+          iconColor="puertoRico"
+        />
+      )}
+      {showBanksLink && (
+        <AppLinkCard slug="banks" icon="bank" iconColor="weirdGreen" />
+      )}
+      {showContactsLink && (
+        <AppLinkCard slug="contacts" icon="team" iconColor="brightSun" />
```

Comment:
Should we create a `ShowLinks` component that handle the several get('data_types') and returns the right components by itself? 

```
<>
 <LaunchTriggerCard initialTrigger={trigger} disabled={isInMaintenance} />
<ShowLinks trigger={trigger}  konnector={konnector} />
</>
```

it'll be easier to maintain, no?

Reply:
I considered it, but since some use `konnector` and some use `trigger`, I thought this abstraction would be too much and actually makes things less maintainable.

Reply:
I am not sure the component approach is best but I think it would be interesting to extract some of the logic from the render, and have it more declarative.

```
const appConfig = {
  banks: {
    shouldBeShown: ({ konnector }) => blabla,
    icon: 'banks'
    iconColor: 'weirdGreen'
  },
  ...
}

const appOrder = ['drive', 'banks', 'contacts']
```

I think it makes the intention clearer and the config can be used elsewhere easily.

Reply:
I'm pretty sure we will add links to linked apps on other pages / components or even apps. 

Having at least a hook that handle the logic to check "bankAccount" or "folder_to_save" or "contacts" can be very useful. 

```
const linkedApps = useLinkedApps({konnector, trigger})

linkedApp.banks && <AppLinkCard slug="banks">
```

Also, I'm wondering if `AppLinkCard` should not be moved to UI with that in mind

Reply:
A mon avis ca devrait surement etre mutualisé avec ce qu il y a ici : https://github.com/cozy/cozy-libs/blob/master/packages/cozy-harvest-lib/src/components/KonnectorSuccess.jsx

Reply:
IMHO, no need for a hook, `useLinkedApps` could be `getLinkedApps` (not really linked ? `getUsefulApps` ?), and this function could use a declarative config.

Reply:
> AMHA that should be mutualized with what already exists for the success message : https://github.com/cozy/cozy-libs/blob/master/packages/cozy-harvest-lib/src/components/KonnectorSuccess.jsx 

I knew it we already have something similar! Thank you!

Reply:
Ah I forgot we had this. Yeah that calls for some form of mutualization then, I'll have a go at this tomorrow.

Reply:
I created a `getRelatedAppsSlugs` that holds only the logic and is used by both `KonnectorSuccess` and `DataTab`. Let me know what you think!